client/src/components/DetailedCard/DetailedCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import MuiDialogTitle from '@material-ui/core/DialogTitle';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Typography from '@material-ui/core/Typography';
+import DetailedCardForm from './DetailedCardForm';
+import { DialogContent } from './useStyles';
+import useStyles from './useStyles';
+import { Box } from '@material-ui/core';
+import EventNote from '@material-ui/icons/EventNote';
+
+interface Props {
+  id: string;
+  children: React.ReactNode;
+  onClose: () => void;
+}
+const DetailedCard: React.FC = () => {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+  const DialogTitle = (props: Props) => {
+    const { children, onClose, ...other } = props;
+    return (
+      <MuiDialogTitle disableTypography className={classes.root} {...other}>
+        <Typography variant="h5" className={classes.cardTitle}>
+          {children}
+          {onClose ? (
+            <IconButton aria-label="close" className={classes.closeButton} onClick={onClose}>
+              <CloseIcon />
+            </IconButton>
+          ) : null}
```

Comment:
You could simply chain the statement and the expression like this :
`!!onClose && ( <IconButton ...` and get rid of returning a `null`

Reply:
Sure I will do that.
Thanks!!